### PR TITLE
[Compute] Upgrading the diskrp sdk to use version 2019-03-01

### PR DESCRIPTION
**Pull request by realschwa**
Completed Swagger PR https://github.com/Azure/azure-rest-api-specs/pull/6824

### CODE REVIEW

**Comment by isra-fel** [comment]
Looks good to me. Just please fix the CI issue.

**Comment by pollock4595** [comment]
@isra-fel the CI tests now pass

File: sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/CreationData.cs [hunk 1]
```diff
@@ -42,13 +42,22 @@ public CreationData()
         /// of a blob to be imported into a managed disk.</param>
         /// <param name="sourceResourceId">If createOption is Copy, this is the
         /// ARM id of the source snapshot or disk.</param>
-        public CreationData(string createOption, string storageAccountId = default(string), ImageDiskReference imageReference = default(ImageDiskReference), string sourceUri = default(string), string sourceResourceId = default(string))
+        /// <param name="sourceUniqueId">If this field is set, this is the
+        /// unique id identifying the source of this resource.</param>
+        /// <param name="uploadSizeBytes">If createOption is Upload, this is
+        /// the size of the contents of the upload including the VHD footer.
+        /// This value should be between 1073741824 (1 GiB + 512 bytes for the
```
Comment (vikramd-ms) [inline sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/CreationData.cs]: This does not look correct. We should be allowing disks of size > 10 MB or so. Please check.

Comment (pollock4595) [reply sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/CreationData.cs]: I thought empty disks and upload disks have a minimum of 1gb?

Comment (pollock4595) [reply sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/CreationData.cs]: PR to fix the description:
https://github.com/Azure/azure-rest-api-specs/pull/7025/commits/9d5e30df16e3ea43cb3b21b5f5264373d8ed2397

**Comment by pollock4595** [comment]
@isra-fel All good to merge now once the CI tests pass

**Review by isra-fel — APPROVED** [review]

**Pull request merged**